webviz_subsurface/plugins/_well_completions.py

```diff
@@ -283,6 +352,31 @@ def create_ensemble_dataset(
     return io.BytesIO(json.dumps(result).encode())
 
 
+def count_leaves(stratigraphy: List[Dict[str, Any]]) -> int:
+    """Counts the number of leaves in the stratigraphy tree"""
+    count = 0
+    for zonedict in stratigraphy:
+        if "subzones" in zonedict:
+            count += count_leaves(zonedict["subzones"])
+        else:
+            count += 1
+    return count
```

Comment:
```suggestion
    return sum(
        count_leaves(zonedict["subzones"]) if "subzones" in zonedict else 1
        for zonedict in stratigraphy
    )
```
I haven't tested if this gives the same result, but think it is possible to shorten it to something like this.

Reply:
Smart! It works :)